airbyte-server/src/main/java/io/airbyte/server/handlers/OAuthHandler.java

```diff
@@ -195,6 +208,9 @@ public Map<String, Object> completeSourceOAuth(final CompleteSourceOauthRequest
 
   public Map<String, Object> completeDestinationOAuth(final CompleteDestinationOAuthRequest completeDestinationOAuthRequest)
       throws JsonValidationException, ConfigNotFoundException, IOException {
+    ApmTraceUtils.addTagsToTrace(Map.of(WORKSPACE_ID_KEY, completeDestinationOAuthRequest.getWorkspaceId(), DESTINATION_DEFINITION_ID_KEY,
+        completeDestinationOAuthRequest.getSourceDefinitionId()));
```

Comment:
```suggestion
        completeDestinationOAuthRequest.getDestinationDefinitionId()));
```

Is this a typo or is this name actually wrong in the class?

Reply:
Typo! Thanks for catching it